ruby/Rakefile

```diff
@@ -24,6 +24,14 @@ proto2_protos = %w[
   google/protobuf/compiler/plugin.proto
 ]
 
+if system('../src/protoc --version')
+  protoc_command = '../src/protoc' 
+elsif system('protoc --version')
+  protoc_command = 'protoc'
+else
+  raise ArgumentError, 'protoc binary not found in src directory or $PATH'
```

Comment:
Our CI test is hitting this error. I'm not really familiar with this Rakefile but based on the comment below I think protoc is not expected to be available when the tests are run in Docker. Maybe it would make sense to just leave `protoc_command` unset (and not raise an error) if the `protoc` binary can't be found?

Reply:
I'll default `protoc_command` to `protoc` so that any compilation failure messages outside Docker will clearly show what the missing binary is.